src/Jira/Api.php

```diff
@@ -574,4 +575,51 @@ public function closeIssue($issueKey)
         }
         return $result;
     }
+    
+    /**
+     * create JIRA Worklog
+     *
+     * @param $issue
+     * @param $filename
+     * @param array $options
+     * @return mixed
+     */
+    public function createWorklog($issueKey, $time, $options = array())
+    {
+        
+        $options = array_merge(
+            array(
+                "timeSpent" => $time,
+            ),
+            $options
+        );
+        
+        return $this->api(self::REQUEST_POST, sprintf("/rest/api/2/issue/%s/worklog", $issueKey), $options);
+    }
+    
+    /**
+     * get JIRA Worklogs for a given issue key
+     *
+     * @param $issueKey 
+     * @return mixed
+     */
+    public function getWorklogs($issueKey)
```

Comment:
Please rebase, because this function was already added as part of #37 and now there is branch conflict that prevents your PR merging.